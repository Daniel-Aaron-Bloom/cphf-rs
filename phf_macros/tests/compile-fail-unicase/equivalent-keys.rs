use unicase::UniCase;
use phf::phf_map;

static MAP: phf::Map<UniCase<&'static str>, isize> = phf_map!( //~ ERROR duplicate key UniCase("FOO")
    UniCase::ascii("FOO") => 42, //~ NOTE one occurrence here
    UniCase::ascii("foo") => 42, //~ NOTE one occurrence here
);

fn main() {}
