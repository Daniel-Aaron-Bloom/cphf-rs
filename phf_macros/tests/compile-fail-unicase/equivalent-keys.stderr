error: unsupported key expression
 --> $DIR/equivalent-keys.rs:5:5
  |
5 |     UniCase("FOO") => 42, //~ NOTE one occurrence here
  |     ^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `proc_macro_call` (in Nightly builds, run with -Z macro-backtrace for more info)
